examples/agent_motion_prediction/agent_motion_config.yaml

```diff
@@ -40,6 +40,9 @@ raster_params:
   # one of the classes we care about (cars, bikes, peds, etc.), >=1.0 filter all other agents.
   filter_agents_threshold: 0.5
 
+  # whether to disable completely traffic light faces in the semantic rasterizer
```

Comment:
Minor: to be exact still had to be 'whether to completely disable traffic light faces in the semantic rasterizer' or 'whether to disable traffic light faces in the semantic rasterizer completely'